src/screens/DiscoveryScreenV2/elements/DiscoveryTab.js

```diff
@@ -4,10 +4,10 @@ import {Dimensions, Pressable, ScrollView, StyleSheet, Text, View} from 'react-n
 import {useRoute} from '@react-navigation/core';
 
 import {Context} from '../../../context';
-import {colors} from '../../../utils/colors';
 import {normalizeFontSize} from '../../../utils/fonts';
 import {setNavbarTitle} from '../../../context/actions/setMyProfileAction';
 import dimen from '../../../utils/dimen';
+import {COLORS} from '../../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `colors` to `COLORS` is a good one as it makes the import statement more consistent with other imports. The new import statement also seems to be more accurate, as it's importing from `utils/theme` instead of `utils/colors`. This change should make the code easier to understand and maintain.
```diff
- import {colors} from '../../../utils/colors';
+ import {COLORS} from '../../../utils/theme';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->